COPYRIGHT

```diff
@@ -0,0 +1,5 @@
+This project is transitioning from an MIT-only license to a dual MIT/Apache-2.0 license.
+Unless otherwise noted, all code contributed prior to 2019-11-21 and not contributed by
+a user listed in [this signoff issue](https://github.com/ipfs/js-ipfs/issues/2552) is
```

Comment:
I will create a separate issue for this - would you be willing to generate and send me a list of contributors to js-ipfs to date so I can mention them in the issue (like in https://github.com/ipfs/go-ipfs/issues/6302)?

Reply:
I will do that.

Reply:
@alanshaw Can you please help me in finding the active IRC or gitter where I can ask questions for js-ipfs. I found few but do not know if they are the right ones.

Reply:
I'm on the freenode IRC \#ipfs